### DISCUSSION TITLE
[Skill Calculator / Firemaking] Pyre logs are missing

### DISCUSSION THREAD

**Discussion by Pirocchio** [runelite/runelite#17770] [Ideas]
I was cremating urium remains earlier today, and noticed that the calculator is missing the cremation process from Firemaking (although it is present in regards to prayer), with the respective toggle from Morytania elite diary 50% bonus xp. 

Also, "making pyre logs (1 dose)" should be there as well, since each dose of sacred oil gives fixed 4 xp.

**Comment by Nightfirecat** [comment]
I've created #17780 for tracking this enhancement.

**Reply by Pirocchio** [reply]
Can I send a last suggestion about this subject before I get 99 fm?

Prayer calculator > urium remains

The calculator states it is 120 xp (180 with Morytania hard diary). The actual values are 120.5 xp (and 180.7).

**Reply by Nightfirecat** [reply]
Please [create a new bug report](https://github.com/runelite/runelite/issues/new?assignees=&labels=needs-triage&projects=&template=bug-issue.yml) for fixing that value.
